docker-compose/6.1.N-docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
 
     transform-router:
         mem_limit: 512m
-        image: quay.io/alfresco/alfresco-transform-router:1.0.2
+        image: quay.io/alfresco/alfresco-transform-router:1.3.2-A2
```

Comment:
ACS 6.1.N  might be one step back too far unless explicitly tested for backwards-compatibility.

I'd start with at least master / ACS 7.0.0 for now (docker-compose.yml) and potentially ACS 6.2.N although should be sanity checked.

Reply:
so, what's your suggestion - revert the change?